service_task_acc.go

```diff
@@ -0,0 +1,63 @@
+package aiven
+
+type (
+	// ServiceTaskHandler Aiven go-client handler for Service tesks
+	ServiceTaskHandler struct {
+		client *Client
+	}
+
+	// ServiceTaskResponse represents service task response
+	ServiceTaskResponse struct {
+		APIResponse
+		Task ServiceTask `json:"task"`
+	}
+
+	// ServiceTask represents a service task
+	ServiceTask struct {
+		CreateTime      string `json:"create_time"`
+		Result          string `json:"result"`
+		TaskType        string `json:"task_type"`
+		Success         bool   `json:"success"`
```

Comment:
Oops already merged before realizing that this should in fact be `*bool`. The value is `null` until the task has completed, after which it will be `false` or `true` depending on whether it succeeded.